### PR TITLE
17 transactions

### LINKED ISSUE #16: Feature: Card list
- [x] Implement fragment with list of cards and ability to swipe between them & change selected card.
- [x] List of curds must be [loaded from server](https://github.com/AlmostGreatBand/lemon-docs#get-cards) using user credentials from issue #15. 
- [x] Cards and last selected card must be cached to local storage.

### LINKED ISSUE #17: Feature: Transaction list
- [x] Implement fragment with transactions of particular cart (passed somehow to fragment).
- [x] List of transactions must be [obtained from server](https://github.com/AlmostGreatBand/lemon-docs#get-transactions).

### PR DESCRIPTION
**Pull request by NeProgramist**
Closes #17 Closes #16

### CODE REVIEW

File: feature-home/src/main/java/com/agb/feature_home/ui/HomeViewModel.kt [hunk 1]
```diff
@@ -3,20 +3,65 @@ package com.agb.feature_home.ui
...
+//            }
+//            cards is Result.Error -> {
+//                _itemsFlow.emit(cards)
+//            }
+//            transactions is Result.Error -> {
+//                _itemsFlow.emit(transactions)
+//            }
+//        }
+//    }
+// }
```
Comment (ALEGATOR1209) [inline feature-home/src/main/java/com/agb/feature_home/ui/HomeViewModel.kt]: Dead code

File: feature-transactions/src/androidTest/java/com/agb/feature_transactions/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.agb.feature_transactions.test", appContext.packageName)
+    }
+}
```
Comment (ALEGATOR1209) [inline feature-transactions/src/androidTest/java/com/agb/feature_transactions/ExampleInstrumentedTest.kt]: Delete

File: feature-transactions/src/test/java/com/agb/feature_transactions/ExampleUnitTest.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
+    @Test
+    fun addition_isCorrect() {
+        assertEquals(4, 2 + 2)
+    }
+}
```
Comment (ALEGATOR1209) [inline feature-transactions/src/test/java/com/agb/feature_transactions/ExampleUnitTest.kt]: Delete

**Review by ALEGATOR1209 — CHANGES_REQUESTED** [review]

File: feature-home/src/main/java/com/agb/feature_home/ui/HomeViewModel.kt (hunk 1 above)
Comment (NeProgramist) [reply feature-home/src/main/java/com/agb/feature_home/ui/HomeViewModel.kt]: not dead, it just waiting it's time
Yoda_SWSB

**Review by ALEGATOR1209 — APPROVED** [review]

**Pull request merged**